pkg/operator/controller/ingress/deployment.go

```diff
@@ -604,6 +606,15 @@ func desiredRouterDeployment(ci *operatorv1.IngressController, ingressController
 		)
 	}
 
+	if ci.Spec.HTTPHeaders != nil && len(ci.Spec.HTTPHeaders.HeaderNameCaseAdjustments) > 0 {
+		var adjustments []string
+		for _, v := range ci.Spec.HTTPHeaders.HeaderNameCaseAdjustments {
+			adjustments = append(adjustments, string(v))
+		}
+		v := strings.Join(adjustments, ",")
```

Comment:
If `Spec.HTTPHeaders.HeaderNameCaseAdjustments` has a length of `1`, the extra comma after the single entry won't break anything, right?

Reply:
Ah, never mind, strings.Join handles that case :smile: